crates/bevy_ecs/src/observer/mod.rs

```diff
@@ -288,6 +288,12 @@ impl Observers {
             world.increment_trigger_id();
             let observers = world.observers();
             let Some(observers) = observers.try_get_observers(event_type) else {
+                #[cfg(feature = "trace")]
```

Comment:
```suggestion
```

Reply:
Do you not want to add a new feature then for debugging? I can add a new feature called `debug` for example and use it here, so it can be used later.

Reply:
I have addressed these nevertheless and if you feel content with its current state this can be merged, but feel free to let me know if you want me to add the feature too in the scope of this work :)

Reply:
I like the idea 🤔 Let's split it out though.